extensions/wikia/MyHome/MyHome.i18n.php

```diff
@@ -99,7 +99,7 @@
 	'activityfeed' => 'Activity feed',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "activityfeed" 38 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)